playbooks/ORCHESTRATION.md

```diff
@@ -229,12 +318,12 @@ This will create a PR signing the snapshot and timestamp files and committed the
 Verify the expirations and the signatures:
 
 ```bash
-./scripts/verify.sh $PR
+$ GITHUB_USER=<your GitHub handle> ./scripts/verify.sh $PR
 ```
 
 Note: You cannot test this step locally against the current staged repository, since the snapshot and timestamp keys are only given permissions to the GitHub Workflows. However, under the hood, the workflow is running `./scripts/step-3.sh` and `./scripts/step-4.sh`. If you initialize a ceremony with local testing keys, this action will work.
 
-## Step 6: Publication
+## Step 7: Publication
 
 Once the PR from [Step 3](#step-3-snapshotting-and-timestamping) is merged, a [workflow](../.github/workflows/sync-ceremony-to-main.yml) will automatically create a PR merging the changes on the completed ceremony branch to main.
 
```

Comment:
Can we remove the extra step under Other? https://github.com/sigstore/root-signing/blob/e728003cce59f5e15c4d2a546e5edad6b76ceff5/playbooks/ORCHESTRATION.md#adding-a-delegation